src/test/java/se/sundsvall/precheck/integration/partyAssets/PartyAssetsClientTest.java

```diff
@@ -0,0 +1,53 @@
+package se.sundsvall.precheck.integration.partyAssets;
+
+
+import generated.client.partyAssets.Asset;
+import generated.client.partyAssets.Status;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.ResponseEntity;
+import org.zalando.problem.Problem;
+import se.sundsvall.precheck.integration.partyAssets.configuration.PartyAssetsIntegration;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
```

Comment:
No wildcard imports

Reply:
Now fixed.

Had to fix bad IDE settings, good catch!